rackcity/utils/errors_utils.py

```diff
@@ -122,3 +124,26 @@ def parse_save_validation_error(error, object_name):
     else:
         failure_detail = object_name + GenericFailure.ON_SAVE.value
     return failure_detail
+
+
+def get_invalid_paginated_request_response(query_params):
```

Comment:
Refactored. This method verifies that the request has the correct query params for a paginated request, returns a 400 JsonResponse if not